index.html

```diff
@@ -190,7 +206,8 @@ <h1>About me</h1>
                 <li>Gitlab</li>
                 <li>Terminal</li>
             
-        </li>
+            </ul>
+    </div>
     </div>
 </section>
 <div class="contact-form">
```

Comment:
- [ ] KIndly align your contact section contents properly to match the Figma design including the texts fonts, color, and alignment 👇🏽 

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/215234041-12b92f6d-0fc5-4b3d-b4ee-c4a654dfcdc6.png)

FIgma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/215234206-87c8be9a-ff64-4d7b-baec-72c5153cd22c.png)